docs/api/browser-window.md

```diff
@@ -1442,15 +1442,16 @@ Sets the properties for the window's taskbar button.
 
 #### `win.setAccentColor(accentColor)` _Windows_
 
-* `accentColor` boolean | string - The accent color for the window. By default, follows user preference in System Settings.
+* `accentColor` boolean | string | null - The accent color for the window. By default, follows user preference in System Settings. To reset to system default, pass `null`.
 
 Sets the system accent color and highlighting of active window border.
 
 The `accentColor` parameter accepts the following values:
 
 * **Color string** - Sets a custom accent color using standard CSS color formats (Hex, RGB, RGBA, HSL, HSLA, or named colors). Alpha values in RGBA/HSLA formats are ignored and the color is treated as fully opaque.
-* **`true`** - Uses the system's default accent color from user preferences in System Settings.
-* **`false`** - Explicitly disables accent color highlighting for the window.
+* **`true`** - Enable accent color highlighting for the window regardless of whether accent colors are currently enabled for windows in System Settings.
+* **`false`** - Disable accent color highlighting for the window regardless of whether accent colors are currently enabled for windows in System Settings.
+* **`null`** - Reset window accent color behavior to follow behavior set in System Settings.
```

Comment:
I think it will be challenging for people to understand the difference between `null` and `true`, but that's probably hard to fix without a breaking change. One thing that might help would be to update the example: https://github.com/electron/electron/blame/6e6509adb6429314b797401bb403b526ac207ea7/docs/api/base-window.md#L1287

Can we update the comment that says:

```
// Use system accent color.
```

Maybe something like:

```
// Enable accent color, using the color specified in System Settings
``` 

Adding an example/comment for `win.setAccentColor(null)` would also be helpful